castai/resource_rebalancing_schedule.go

```diff
@@ -250,8 +283,18 @@ func scheduleToState(schedule *sdk.ScheduledrebalancingV1RebalancingSchedule, d
 		"node_ttl_seconds":   schedule.LaunchConfiguration.NodeTtlSeconds,
 		"num_targeted_nodes": schedule.LaunchConfiguration.NumTargetedNodes,
 	}
+
 	if schedule.LaunchConfiguration.RebalancingOptions != nil {
 		launchConfig["rebalancing_min_nodes"] = schedule.LaunchConfiguration.RebalancingOptions.MinNodes
+
+		if schedule.LaunchConfiguration.RebalancingOptions.ExecutionConditions != nil {
```

Comment:
introduce a temporary variable for `schedule.LaunchConfiguration.RebalancingOptions.ExecutionConditions` to reduce copy/paste in the mapping